mobile/src/people/current-people-department.tsx

```diff
@@ -33,14 +37,15 @@ class CurrentPeopleDepartmentImpl extends React.Component<CurrentPeopleDepartmen
 
     //----------------------------------------------------------------------------
     public static navigationOptions: NavigationScreenConfig<NavigationStackScreenOptions> = (navigationOptionsContainer) => {
+        return navigationOptionsWithTitle(navigationOptionsContainer, CurrentPeopleDepartmentImpl.getTitle(navigationOptionsContainer));
+    };
+
+    //----------------------------------------------------------------------------
+    private static getTitle(navigationOptionsContainer: NavigationOptionsContainer): string {
         const navigation = navigationOptionsContainer.navigation;
         const departmentAbbreviation = navigation.getParam('departmentAbbreviation', undefined);
```

Comment:
Default parameter can be used instead of undefined and extra check in return won't be needed

Reply:
Done. e1ccebad9c46778a8e340a1f5380e551730dae2b